addons/sale/report/sale_report_templates.xml

```diff
@@ -67,86 +67,105 @@
             </div>
 
             <!-- Is there a discount on at least one line? -->
-            <t t-set="display_discount" t-value="any([l.discount for l in doc.order_line])"/>
+            <t t-set="display_discount" t-value="any([line.discount for line in doc.order_line])"/>
             <!-- if Rounding Method is computed per line -->
-            <t t-set="display_taxamount"
-               t-value="doc.company_id.tax_calculation_rounding_method == 'round_per_line'"/>
-
-            <t t-foreach="doc.order_lines_layouted()" t-as="page">
-                <table class="table table-condensed">
-                    <thead>
-                        <tr>
-                            <th>Description</th>
-                            <th class="text-right">Quantity</th>
-                            <th class="text-right">Unit Price</th>
-                            <th t-if="display_discount" class="text-right" groups="sale.group_discount_per_so_line">Disc.(%)</th>
-                            <th class="text-right">Taxes</th>
-                            <th t-if="display_taxamount" class="text-right">Tax Amount</th>
-                            <th class="text-right" groups="sale.group_show_price_subtotal">Amount</th>
-                            <th class="text-right price_tax_included" groups="sale.group_show_price_total">Total Price</th>
-                        </tr>
-                   </thead>
-                   <tbody class="sale_tbody">
-                        <t t-foreach="page" t-as="layout_category">
-
-                            <t t-if="layout_category_size > 1 or page_size > 1" groups="sale.group_sale_layout">
-                                <tr class="active">
-                                    <td colspan="7" style="font-weight: bold; border-bottom: 1px solid black;">&amp;bull;
-                                        <t t-esc="layout_category['name']"/>
-                                    </td>
-                                </tr>
-                            </t>
+            <t
```

Comment:
Small note: unnecessary diff (there are probably other unnecessary diff, you could go through it and try to remove everything that is not necessary to lessen the diff)

Reply:
Reverted.